packages/shopify-app-session-storage-mysql/src/mysql.ts

```diff
@@ -151,17 +165,31 @@ export class MySQLSessionStorage implements SessionStorage {
           accessToken varchar(255)
         )
       `;
-      await this.query(query);
+      await this.connection.query(query);
     }
   }
 
-  private query(sql: string, params: any[] = []): Promise<any> {
-    return this.connection.query(sql, params);
-  }
-
   private databaseRowToSession(row: any): Session {
     // convert seconds to milliseconds prior to creating Session object
     if (row.expires) row.expires *= 1000;
     return Session.fromPropertyArray(Object.entries(row));
   }
+
+  private async initMigrator(
+    migratorOptions?: SessionStorageMigratorOptions | null,
```

Comment:
REALLY nit-y: if we already allow `migratorOptions` to be optional, do we need the `null` value as well? We could just either get a well-formed object, or nothing at all (i.e. `undefined`), right?